src/PeriodEnd/Refunds/SFA.DAS.ProviderPayments.Calc.Refunds/SFA.DAS.ProviderPayments.Calc.Refunds.csproj

```diff
@@ -67,6 +67,7 @@
     <Compile Include="DependencyResolution\RefundsRegistry.cs" />
     <Compile Include="Domain\RefundGroup.cs" />
     <Compile Include="Dto\AccountLevyCredit.cs" />
+    <Compile Include="Dto\HistoricalPayment.cs" />
```

Comment:
Can you move it to the Domain folder

Reply:
I'm confused on the difference between the domain folder, and the dto folder. Happy to do this but would like to discuss the purpose of each of these folders.